server/service.go

```diff
@@ -544,6 +566,8 @@ func (m *BoostService) handleGetPayload(w http.ResponseWriter, req *http.Request
 	// Wait for all requests to complete...
 	wg.Wait()
 
+	// send bid and signed block to relay monitor
+	m.sendAuctionTranscriptToRelayMonitors(AuctionTranscript{Bid: originalBid.response.Data, Acceptance: payload})
```

Comment:
is there a way to return early before this is fired?

`mev-boost` should post this data to any relay monitors if it gets the data called `payload` here, regardless of any relay behavior -- and there is a large chunk of code to manage that leg of this journey in between the start of this function and making the call to the relay monitor

Reply:
yeah. this function does just spawn off a few more goroutines, but that could change in the future, so it'd be better to keep the `go m.sendAuctionTranscriptToRelayMonitors` here.